src/main/python/README.md

```diff
@@ -1,4 +1,7 @@
-# Python Readme
+# SystemDS
+
+[![Build Status](https://travis-ci.com/tugraz-isds/systemds.svg?branch=master)](https://travis-ci.com/tugraz-isds/systemds)
+[![License](https://img.shields.io/badge/License-Apache%202.0-gre.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
The Travis is pointing at out old Repository, and more importantly, the Travis test only execute component test which does not effect the python part.
Look at the main README, and maybe copy the python badges?

Reply:
Should I keep the build status? or just license and python?

Reply:
I decided on just the python test.

Reply:
@kev-inn - apache projects doesnot need a license badge because of the `apache`. Shall we just remove apache license badge. 

If this is okay for you,  I will remove this one badge before commiting.

Reply:
then remove the one we have in our main readme as well.